flaskapp/resources/utils.py

```diff
@@ -0,0 +1,17 @@
+class InvalidCoordinates(BaseException):
```

Comment:
It's a nit, but this whole file is tabbed to 6 spaces, where most others are 4 spaces.